analyzer/consensus/static/accounts.go

```diff
@@ -17,16 +17,32 @@ import (
 //go:embed accounts
 var genesisAccountsFs embed.FS
 
+//go:embed delegations
+var genesisDelegationsFs embed.FS
```

Comment:
would it make sense to move the new code into the `consnesus/static/delegations.go` file